From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 29 Sep 2009 13:36:19 +0200
Subject: [PATCH] Use gtk_window_present instead of gtk_show_widget

so the dialog is pushed into the foreground when selected from the menu.
---
 src/krb5-auth-tickets.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/krb5-auth-tickets.c b/src/krb5-auth-tickets.c
index b00687c..7c177a9 100644
--- a/src/krb5-auth-tickets.c
+++ b/src/krb5-auth-tickets.c
@@ -94,7 +94,7 @@ void
 ka_tickets_dialog_run()
 {
 	if (ka_get_service_tickets(tickets)) {
-		gtk_widget_show(tickets_dialog);
+		gtk_window_present(GTK_WINDOW(tickets_dialog));
 		gtk_dialog_run(GTK_DIALOG(tickets_dialog));
 		gtk_widget_hide(tickets_dialog);
 	} else {
-- 
